vnext/docs/api/react-native-windows.iviewwindowsprops.md

```diff
@@ -10,6 +10,12 @@
 export interface IViewWindowsProps extends IKeyboardProps, ViewProps 
 ```
 
+## Remarks
+
+Props type for ViewWindows component
+
+Extends: [IKeyboardProps](./react-native-windows.ikeyboardprops.md) and [react-native's ViewProps](https://facebook.github.io/react-native/docs/view)
```

Comment:
Not sure why this file is changed.. will remove.

Reply:
Someone should check this change in.. its auto generated, and so is probably showing up in everyones builds currently.

Reply:
I see, will leave it in the change.,

---
In reply to: [336108926](https://github.com/microsoft/react-native-windows/pull/3447#discussion_r336108926) [](ancestors = 336108926)